src/azure-cli/azure/cli/command_modules/appservice/_constants.py

```diff
@@ -95,4 +95,4 @@ def __init__(self):
     'powershell': 'FunctionApp/windows-powershell-functionapp-on-azure.yml',
 }
 
-DEFAULT_CENTAURI_IMAGE = 'azurefunctionstest.azurecr.io/azure-functions/dotnet7-quickstart-demo:1.0'
+DEFAULT_CENTAURI_IMAGE = 'mcr.microsoft.com/azure-functions/dotnet7-quickstart-demo:1.0'
```

Comment:
I'd like to confirm that the position of the default image has changed, but the image itself has not changed, right? It won't cause a breaking change for customers, will it?

Reply:
Yes, the image itself has not changed. This default image has only been used by private preview customers for Centauri so even if it is a breaking change, we can communicate that they update the image if they were using the previous default image.

Reply:
Got it, thanks